.github/workflows/tests.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     - name: Build opendbc
       run: ${{ env.RUN }} "cd ../ && scons -j$(nproc) --minimal"
     - name: Unit tests
-      run: ${{ env.RUN }} "pytest"
+      run: ${{ env.RUN }} "pytest -n auto"
```

Comment:
openpilot uses -n logical for hyperthreading I believe

Reply:
Normally it uses auto: https://github.com/commaai/openpilot/blob/master/pyproject.toml#L139
but in CI we use logical: https://github.com/commaai/openpilot/blob/master/.github/workflows/selfdrive_tests.yaml#L30